test/LeanCode.IntegrationTests/App/ApiModule.cs

```diff
@@ -48,7 +48,7 @@ public override void ConfigureServices(IServiceCollection services)
             })
             .AddJwtBearer(cfg =>
             {
-                cfg.Authority = UrlHelper.Concat(config.GetValue<string>("InternalBase"), "auth");
+                cfg.Authority = UrlHelper.Concat(config.GetValue<string>("InternalBase") ?? "", "auth");
```

Comment:
```suggestion
                cfg.Authority = UrlHelper.Concat(config.GetValue<string>("InternalBase")!, "auth");
```

It should always be there.